src/ToricDivisors.jl

```diff
@@ -1,46 +1,76 @@
-include("ToricVarieties.jl")
-
-
 ######################
 # 1: The Julia type for ToricDivisors
 ######################
 
-struct toric_divisor
+struct toricDivisor
```

Comment:
So, I am realllly sorry for all the churn and work I cause you... but really the convention in Julia is to use `CamelCase` for structs, not `camelCase`...

Reply:
Oh... This one is on me. You wrote `CamelCase` above - I should have read carefully. Hence, no worries!

Since this PR is already terribly long (and you just approved), I opened a new issue: https://github.com/oscar-system/JToric.jl/issues/44.

This one should be closed with the adjustment to `CamelCase` soon.